docs/proposals/initial-resources.md

```diff
@@ -38,11 +38,10 @@ and set them before the container is run. This document describes design of the
 
 ## Motivation
 
-Since we want to make Kubernetes as simple as possible for its users we don’t want to require setting
-[Resources](resource-qos.md#resource-specifications)
-for container by its owner. On the other hand having Resources filled is critical for scheduling decisions.
-Current solution to set up Resources to hardcoded value has obvious drawbacks. We need to implement a component
-which will set initial Resources to a reasonable value.
+Since we want to make Kubernetes as simple as possible for its users we don’t want to require setting [Resources](../design/resource-qos.md) for container by its owner.
+On the other hand having Resources filled is critical for scheduling decisions.
+Current solution to set up Resources to hardcoded value has obvious drawbacks.
+We need to implement a component which will set initial Resources to a reasonable value.
```

Comment:
This is already implemented as of 1.1. Unfortunately there doesn't seem to be any user or admin documentation yet, just design docs in proposals/. I've asked the autoscaling team to write something, and then we can link to it here.